vm/systemSmartContracts/validator.go

```diff
@@ -1370,6 +1442,10 @@ func (v *validatorSC) unBondTokens(args *vmcommon.ContractCallInput) vmcommon.Re
 	if returnCode != vmcommon.Ok {
 		return returnCode
 	}
+	if v.isUnStakeUnBondPaused() {
```

Comment:
is this check necessary on the unbond also? The nodes were unstaked at least 10 days ago.

Reply:
yes. it is important to stop unBond as well.

Reply:
tokens can't leave the contract in case of pause